apm-sniffer/apm-sdk-plugin/sharding-jdbc-1.5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/sjdbc/ExecuteEventListener.java

```diff
@@ -62,13 +62,17 @@ private void handle(AbstractExecutionEvent event, String operation) {
                 AbstractSpan span = ContextManager.createExitSpan("/SJDBC/BRANCH/" + operation, event.getDataSource());
                 if (ExecutorDataMap.getDataMap().containsKey(AsyncExecuteInterceptor.SNAPSHOT_DATA_KEY)) {
                     ContextManager.continued((ContextSnapshot) ExecutorDataMap.getDataMap()
-                                                                              .get(AsyncExecuteInterceptor.SNAPSHOT_DATA_KEY));
+                        .get(AsyncExecuteInterceptor.SNAPSHOT_DATA_KEY));
                 }
                 Tags.DB_TYPE.set(span, "sql");
                 Tags.DB_INSTANCE.set(span, event.getDataSource());
                 Tags.DB_STATEMENT.set(span, event.getSql());
                 if (!event.getParameters().isEmpty()) {
-                    Tags.DB_BIND_VARIABLES.set(span, Joiner.on(",").join(event.getParameters()));
+                    String variables = event.getParameters()
```

Comment:
What thing is NULL triggering NPE?

Reply:
event.getParameters contains null value,inside Joiner will check null value and then throw NPE

Reply:
event.getParameters contains null value,inside Joiner will check null value and then throw NPE

Reply:
event.getParameters contains null value,inside Joiner will check null value and then throw NPE

Reply:
Got it. LGTM.